stream/src/test/java/org/apache/apex/malhar/stream/FunctionOperator/FunctionOperatorTest.java

```diff
@@ -1,332 +0,0 @@
-/**
```

Comment:
As I said before this is due to conflicts and merging from upstream that show up in my branch. Will figure out a way to avoid this in future.